cocos/scene-graph/node.ts

```diff
@@ -1986,14 +1988,9 @@ export class Node extends CCObject implements ISchedulable, CustomSerializable {
                 if (dirtyBits & TransformBit.RS) {
                     Mat4.fromRTS(child._mat, child._lrot, child._lpos, child._lscale);
                     Mat4.multiply(child._mat, cur._mat, child._mat);
-                    if (dirtyBits & TransformBit.ROTATION) {
-                        Quat.multiply(child._rot, cur._rot, child._lrot);
-                    }
```

Comment:
child._rot is effected by parent's scale,  `child.worldRotation != parent.worldRotation * child.localRotation"